static/src/stylesheets/head.content.scss

```diff
@@ -24,6 +24,7 @@
 @import 'module/_chapters';
 @import 'module/content/_article-immersive';
 @import 'module/content/_article-minute';
+@import 'module/content/_article-labour-liverpool';
```

Comment:
Good shout. Would this involve adding something to this object:

https://github.com/guardian/frontend/blob/master/common/app/assets/assets.scala#L51